crates/protocol/src/batch/validity.rs

```diff
@@ -0,0 +1,53 @@
+//! Contains the [BatchValidity] and its encodings.
+
+/// Batch Validity
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum BatchValidity {
+    /// The batch is invalid now and in the future, unless we reorg
```

Comment:
if i'm understanding correctly

```suggestion
    /// The batch is invalid now and in the future, unless we reorg, so it can be discarded.
```